components/Layout.js

```diff
@@ -51,53 +51,129 @@ export default class Layout extends React.Component {
             font-family: 'Lora';
             font-size: 18px;
           }
-
+          p {
```

Comment:
Maybe you can put all this css in its own file and [import](https://codeburst.io/4-four-ways-to-style-react-components-ac6f323da822) it into this layout.

Reply:
Just moved this to an external file - lmk what you think!